rfq/negotiator.go

```diff
@@ -281,6 +299,26 @@ func (n *Negotiator) HandleIncomingBuyRequest(
 	go func() {
 		defer n.Wg.Done()
 
+		// If the negotiator is configured to accept all prices, then we
+		// will skip querying the price oracle and accept the price
+		// provided by the peer.
+		if n.cfg.AcceptAllPrices {
+			// TODO(ffranr): The expiry time should be provided
+			//  by the peer in the buy request message.
+			currentTime := time.Now().Unix()
+			expiryTimestamp := uint64(
+				currentTime + defaultRateTickExpirySeconds,
+			)
+
+			// Construct and send a sell accept message.
+			msg := rfqmsg.NewBuyAcceptFromRequest(
+				request, request.BidPrice, expiryTimestamp,
+			)
+			sendOutgoingMsg(msg)
+
+			return
+		}
```

Comment:
This should auto reject if `AcceptAllPrices` isn't specified. So we want 3 cases: 
  * no oracle specified, reject (default config) 
  * accept all 
  * query the oracle